applications/ndk/1.3.0/release/ndk.yaml

```diff
@@ -0,0 +1,34 @@
+apiVersion: source.toolkit.fluxcd.io/v1
+kind: OCIRepository
+metadata:
+  name: ndk
+  namespace: ${releaseNamespace}
+spec:
+  interval: 1m 
+  url: "${ociRegistryURL:=oci://ghcr.io}/mesosphere/charts/ndk"
```

Comment:
what is `ociRegistryURL` and who is populating that value - I dont think its populated by any component.

Reply:
@kshitij-kumar-ngt ^